iox_query/src/provider/physical.rs

```
@@ -190,6 +193,18 @@ pub fn chunks_to_physical_nodes(
         let output_ordering =
             sort_key.map(|sort_key| arrow_sort_key_exprs(&sort_key, &file_schema));
 
+        let props = ExecutionProps::new();
+        let filter_expr = predicate.filter_expr()
+            .and_then(|filter_expr| {
+                match create_physical_expr_from_schema(&props, &filter_expr, &file_schema) {
+                    Ok(f) => Some(f),
+                    Err(e) => {
+                        warn!(%e, ?filter_expr, "Error creating physical filter expression, can not push down");
```

this is consistent with what PruningPredicate used to do (ignore errors and simply not prune with such expressions) - required due to https://github.com/apache/arrow-datafusion/pull/5419